forch/faucetizer.py

```diff
@@ -225,13 +225,15 @@ def _initialize_host_ports(self, behavioral_faucet_config):
     def _finalize_host_ports_config(self, behavioral_faucet_config, new_testing_device_vlans):
         testing_port_vlans = new_testing_device_vlans.values()
         testing_port_configured = False
+        should_apply_tail_acl = self._config.tail_acl and self.validate_tail_acl_config()
```

Comment:
Correct me if I have this wrong:
A = self._config.tail_acl; B = self._has_acl(self._config.tail_acl); A' = not self._config.tail_acl;

validate_tail_acl_config = A' or B

should_apply_tail_acl = A and (A' or B)
                                      = (A and A') or (A and B)
                                      = None or (A and B).  = A and B

Logically, would it be the same thing to simplify should_apply_tail_acl = self._config.tail_acl and self._has_acl(self._config.tail_acl)

Reply:
My bad. Just checked comments above.

Reply:
I see..